### ISSUE TITLE
Setup failed since latest Google patch (June)

### ISSUE THREAD

**Issue by HarriBuh** [Magisk-Modules-Alt-Repo/open_webview#39]
[magisk_install_log_2024-06-12T20.24.58.log](https://github.com/user-attachments/files/15809425/magisk_install_log_2024-06-12T20.24.58.log)
```
!!! Overlay creation has failed !!!
Compatibility is unlikely, please report this to your ROM developer.
Some ROMs need a patch to fix this.
Do NOT report this issue to me.

Aborting...
! Installation failed
```

Stock Android 14
Google Pixel 7
Magisk 27.0

I DID report this to you, since Google won't change or do anything about it and the only chance for it to start working again would be YOU helping me to find a fix.

And, most importantly, your module has always been the only working way to install a custom webview on my phone.

**Comment by Cgritsch89** [comment]
The same problem I have please update it

**Comment by Cgritsch89** [comment]
Thanks for the solution, now to a other question, how I can update the mulch webview?

**Comment by porsche613** [comment]
Same problem. After Webview installed, prompt "!!! Overlay creation has failed !!!".
Lineage for microG base on Lineage OS 21 ( 20240817 ), GM1911.

The overlays apk installation has failed, prompt "not compatible". I built the overlay apk, and even put the overlay apk into the /system/product/overlay/ it won't be dexed by the system. It's interesting.

By the way, LSPosed ( [Latest Official 1.9.2](https://github.com/LSPosed/LSPosed/releases/tag/v1.9.2) and [Android 15b4 supported fork](https://github.com/JingMatrix/LSPosed) ) and [AnyWebView](https://github.com/neoblackxt/AnyWebView) is still working confirmed.

**Comment by F3FFO** [comment]
Thanks to @XRevan86 this is fixed in 2.5.0
